lib/dpl/provider/bintray.rb

```diff
@@ -0,0 +1,505 @@
+require 'json'
+require 'net/http'
+require "uri"
+require 'find'
+
+module DPL
+  class Provider
+    class Bintray < Provider
+      @test_mode = false
+
+      def check_auth
+      end
+
+      def needs_key?
+        false
+      end
+
+      attr_writer :test_mode
+
+      def init_from_args
```

Comment:
I'm a bit confused by this method. Why is the initialization code not in `#initialize`?